software/donetick.yml

```
@@ -0,0 +1,17 @@
+name: "Donetick"
+website_url: "https://donetick.com"
+source_code_url: "https://github.com/donetick/donetick"
+description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Nodejs
+  - Docker
```

Nodejs is not needed for installation, therefore can be excluded
```suggestion
platforms:
  - Go
  - Docker
```